### PR TITLE
Tweak: Remove Core/Pro dependencies in version 3.27 [ED-15717]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: core/experiments/manager.php [hunk 1]
```diff
@@ -439,19 +439,6 @@ private function add_default_features() {
 			'release_status' => self::RELEASE_STATUS_ALPHA,
 			'default' => self::STATE_ACTIVE,
 		] );
-
-		// TODO: Possibly remove experiment in v3.27.0 [ED-15717].
```
Comment (hein-obox) [inline core/experiments/manager.php:443]: @rami-elementor I believe it is safe to remove this experiment, as it was hidden and set to immutable for 2 versions already.

Comment (rami-elementor) [reply core/experiments/manager.php:443]: I agree

**Review by rami-elementor — DISMISSED** [review]

**Review by Batsirai-muchareva — DISMISSED** [review]

**Comment by louiswol94** [comment]
Regarding deleting the `global-css.php` file, I know we should remove it at some point and we only support two versions, but maybe we should push it out even longer... The issue is that it triggers a fatal error on version upgrade if the file is not found. WDYT?

**Comment by hein-obox** [comment]
> Regarding deleting the `global-css.php` file, I know we should remove it at some point and we only support two versions, but maybe we should push it out even longer... The issue is that it triggers a fatal error on version upgrade if the file is not found. WDYT?

@louiswol94 Can you please guide us? This time, the Playwright tests seem to be passing. I don't really understand why we got the errors in the past.
What is your proposal?
Should we create a separate task for this?

**Comment by louiswol94** [comment]
> > Regarding deleting the `global-css.php` file, I know we should remove it at some point and we only support two versions, but maybe we should push it out even longer... The issue is that it triggers a fatal error on version upgrade if the file is not found. WDYT?
> 
> @louiswol94 Can you please guide us? This time, the Playwright tests seem to be passing. I don't really understand why we got the errors in the past. What is your proposal? Should we create a separate task for this?

@hein-obox I just saw that with older versions it throws an error. And because it's a fatal error, I was wondering how many users are on older (than 2) versions. And maybe we should keep it a bit longer until most of our users are on 3.24/3.25 - can't remember which version removed the references to this file.

**Comment by hein-obox** [comment]
> > > Regarding deleting the `global-css.php` file, I know we should remove it at some point and we only support two versions, but maybe we should push it out even longer... The issue is that it triggers a fatal error on version upgrade if the file is not found. WDYT?
> > 
> > 
> > @louiswol94 Can you please guide us? This time, the Playwright tests seem to be passing. I don't really understand why we got the errors in the past. What is your proposal? Should we create a separate task for this?
> 
> @hein-obox I just saw that with older versions it throws an error. And because it's a fatal error, I was wondering how many users are on older (than 2) versions. And maybe we should keep it a bit longer until most of our users are on 3.24/3.25 - can't remember which version removed the references to this file.

@louiswol94 Shall we for safety, postpone this removal to v3.28?

**Review by louiswol94 — APPROVED** [review]

**Pull request merged**